networkx/algorithms/bipartite/tests/test_project.py

```diff
@@ -66,6 +69,12 @@ def test_path_weighted_projected_graph(self):
         assert edges_equal(list(P.edges()), [(0, 2)])
         P[0][2]["weight"] = 1
 
+    def test_digraph_conversion(self):
```

Comment:
Maybe a better test name would be: ???
```suggestion
    def test_digraph_weighted_projection(self):
```